dstep/translator/Output.d

```diff
@@ -702,8 +704,15 @@ D"[0 .. $ - 1]);
 
             foreach (line; lines)
             {
-                indent();
-                buffer.put(line[indentAmount .. $]);
+                if (line.all!isWhite)
```

Comment:
Do we need to deal with Unicode whitespace? I assume `isWhite` is from the `std.ascii` module and that it doesn't deal with Unicode whitespace.

Reply:
There's a `isWhite` function in `std.uni` as well.

Reply:
"indentAmount" is counted as number of the spaces, so the code should be correct in the sense it shouldn't crash anymore.

Regarding std.uni it's big pain to use as it works on dchar and string consists of utf8 `char`s.

Reply:
The unicode library in D isn't the wisest design : (